src/Apps/Consign/Routes/MarketingLanding/Components/SoldRecentlyOnArtsy.tsx

```diff
@@ -58,29 +52,32 @@ export const SoldRecentlyOnArtsy: React.FC<SoldRecentlyOnArtsyProps> = ({
             if (!artwork) return <></>
 
             return (
-              <Fragment key={artwork.internalID}>
-                <ShelfArtworkFragmentContainer
-                  artwork={artwork}
-                  hideSaleInfo
-                  lazyLoad
-                  width={[325]}
-                  data-testid="soldRecentlyItem"
-                  onClick={trackArtworkItemClick(artwork, i)}
-                  // FIXME:
-                  contextModule={ContextModule.artworkRecentlySoldGrid as any}
-                />
+              <ShelfArtworkFragmentContainer
+                key={artwork.internalID}
+                artwork={artwork}
+                hideSaleInfo
+                lazyLoad
+                area={60000}
```

Comment:
Hunch is that this is gonna be kinda hard for devs to remember / understand, at least initially. Can you break it down a bit for me just so I understand better? (once i understand completely we might want to come up with a better prop name.. maybe)

Reply:
I wouldn't have exposed it otherwise.

Reply:
Still, might be good to make the name more clear... `maxHorizontalArea` or something. "Area" is so vague! (No strong feelings about it tho other than "whoa, what does area do", which I don't know)

Reply:
I agree it's weird but it's not a maxHorizontalArea. It's 'how much area do u want this image to occupy?'

Reply:
Oh I see. Feels weird no matter how you describe it. I've got no naming ideas other than to suggest there's a good comment on the interface describing its function; `area={60000}` is foreign and indiscernible in terms of function, and whats that super large number? Anywaysss

Reply:
Yeah like 'totalNumberOfPixelsInImage' isn't a ton better.

Reply:
right